rest-api/src/main/resources/definitions/study.yml

```diff
@@ -14,11 +15,52 @@ properties:
         type: string
         description: The name of the study.
         x-nullable: false
+    details:
+        type: string
+        description: An in-depth description of this study for prospective participants, or to show participants after they have joined a study (the “about” screen for the study).
+    phase:
+        $ref: ./enums/study_phase.yml
+        description: The lifecycle phase this study is in.
+        x-nullable: false
     clientData:
         type: object
         description: |
             Client data for a study should be in a syntactically valid JSON format. It will be returned as is to the client (as JSON). It can contain any information that is useful to study authoring tools or administration tools that is not part of the Bridge data model.
         additionalProperties: true
+    irbApprovedOn:
+        type: string
+        description: Before the study can launch, the IRB’s approval must be recorded for the study.
+        format: date 
+    irbApprovedUntil:
+        type: string
+        description: Before the study can launch, the IRB’s approval must be recorded for the study.
+        format: date 
```

Comment:
Does this generate a LocalDate value?

Reply:
The Swagger UI tool correctly puts a YYYY-MM-DD value, but I'll add an example to all the format: date entries, it's harmless. I'm also adding the value to the model browser so it appears there.